static/js/sefaria/sefaria.js

```diff
@@ -1046,6 +1046,15 @@ Sefaria = extend(Sefaria, {
       }
       return ref ? this.getRefFromCache(ref) : null;
   },
+  transCallToActionApplies: (book, textLanguage) => {
```

Comment:
The idea was that we tend to put functions that purely deal with data (as opposed to rendering) is sefaria.js. When we need to take into account existing translations, I imagine it would either need to do an API call or check the versions cache for some new data we're sending.